src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -16,7 +16,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-"""Plugin for Sphinx.
+"""Sphinx-plugin for PyBuilder to make a sphinx-quickstart and sphinx-generate_documentation.
```

Comment:
Maybe reformulate to "to run a sphinx quickstart and generate the documentation once set up" ?